app/controllers/account/calculators_controller.rb

```diff
@@ -1,7 +1,6 @@
 # frozen_string_literal: true
 
 class Account::CalculatorsController < Account::BaseController
-  before_action :resource, only: [:edit, :update, :destroy]
   load_and_authorize_resource
```

Comment:
maybe we should skip "load_and_authorize_resource", because we load resources and have only user aka Admin.